dev-tools/import_dashboards.sh

```diff
@@ -171,10 +171,10 @@ fi
 if [ -d "${DIR}/index-pattern" ]; then
     for file in ${DIR}/index-pattern/*.json
     do
-        NAME=`awk '$1 == "\"title\":" {gsub(/[",]/, "", $2); print $2}' ${file}`
```

Comment:
@monicasarbu I changed it as the code did not work anymore. I changed it now to `${NAME}-*`. This is only a temporary fix and this script should be removed when https://github.com/elastic/beats/pull/2155 is merged.